### PR TITLE
Blog rss fix

### PR DESCRIPTION
**Pull request by tomwilkie**

### CODE REVIEW

**Comment by tomwilkie** [comment]
DO NOT MERGE.  This is failing, and I'm trying to get the build to fail correctly.

File: layouts/blog/single.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 		<h1 class="page-letterhead__heading page-letterhead__heading--on-background">{{ .Title }}</h1>
 		<h2 class="page-letterhead__subheading">
 			Published: {{ .Date.Format "2 Jan 2006" }} by {{ $author.name }}
-			<a class="blog-rss-link" href="{{ .RSSLink }}" type="application/rss+xml" target="_blank">
+			<a class="blog-rss-link" href="{{ Site.RSSLink }}" type="application/rss+xml" target="_blank">
```
Comment (cinaglia) [inline layouts/blog/single.html]: This is missing a prefix `.`

Comment (DanCech) [inline layouts/blog/single.html]:
```suggestion
			<a class="blog-rss-link" href="{{ .Site.RSSLink }}" type="application/rss+xml" target="_blank">
```

Comment (torkelo) [reply layouts/blog/single.html]: thanks, not sure how that happend, sure I had a dot there when I tested & deployed to staging.

File: Gruntfile.js [hunk 2]
```diff
@@ -231,7 +231,7 @@ module.exports = function(grunt) {
 
     hugo = require("child_process").spawn("hugo", args, {stdio: "inherit"});
     hugo.on("exit", function() { done(true); });
-    hugo.on("error", function() { done(true); });
+    hugo.on("error", function() { done(false); });
```
Comment (cinaglia) [inline Gruntfile.js]: The `error` event will not be triggered if the process was spawned successfully. What you want here is to hook into `exit` and check whether the exit code is 0. Something like:

```js
hugo.on("exit", function(code) { done(code !== 0) });
```

**Comment by torkelo** [comment]
shall we try to merge this now that we have fail on error & RSSLink syntax fix

File: Gruntfile.js (hunk 2 above)
Comment (DanCech) [reply Gruntfile.js]: Is code 0 success or failure?

Comment (cinaglia) [reply Gruntfile.js]: 0 is success. The example was incorrect. I just pushed a commit that should fix it.

**Comment by torkelo** [comment]
Giving this a shot now

**Pull request merged**